src/Interpreters/executeQuery.cpp

```diff
@@ -234,6 +240,10 @@ static void onExceptionBeforeStart(const String & query_for_logging, Context & c
 
     elem.current_database = context.getCurrentDatabase();
     elem.query = query_for_logging;
+    elem.normalized_query_hash = normalizedQueryHash(query_for_logging);
+
+    // We don't calculate query_kind, databases, tables and columns when the query isn't able to start
```

Comment:
@amosbird hi, why not we calcute this information? example: select count(1) from numbers(10) where number in url('xxx', 'xxx', 'xxx'), when this url return code is 403. It cann't get the tables from system.query_log.

Reply:
Ignore. The lasted of version not existes this problem